src/api.ts

```diff
@@ -67,4 +68,15 @@ router.get('/spaces/:key/poke', async (req, res) => {
   }
 });
 
+router.post('/flag', verifyAuth, async (req, res) => {
+  const { type, value, action } = req.body;
+
+  try {
+    await flagEntity({ type, value, action });
+    return res.json({ success: true });
```

Comment:
This action always return true, whether the space/proposal exist or not. Maybe use the `affectedRows` from the query result to return whether the sql query has succeed or not ?

Reply:
We will use this endpoint inside the laser. And previously laser hasn’t handled this scenario before (you were able to insert the wrong values). I just followed the previous approach we had